.github/workflows/python.yml

```diff
@@ -145,8 +145,29 @@ jobs:
         with:
           packages-dir: python/foxglove-schemas-protobuf/dist
 
+  notebook-fe:
```

Comment:
nit: I find the "fe" abbreviation a bit confusing, would prefer "frontend" throughout